src/codegen/general.rs

```diff
@@ -16,32 +16,71 @@ use std::{
 
 pub fn start_comments(w: &mut dyn Write, conf: &Config) -> Result<()> {
     if conf.single_version_file.is_some() {
-        return start_comments_no_version(w);
+        return start_comments_no_version(w, conf);
     }
-    writeln!(
+    write!(
```

Comment:
You could possibly just call `single_version_file` and save some duplication. (Perhaps rename the function as well)